plugin/storage/es/spanstore/reader.go

```diff
@@ -83,20 +85,33 @@ var (
 	tagFieldList = []string{tagsField, processTagsField, logFieldsField}
 )
 
+type spanReaderMetrics struct {
```

Comment:
https://github.com/uber/jaeger/blob/master/storage/spanstore/metrics/decorator.go

don't know, maybe we only use it internally. Please verify that this decorator captures the same metrics